### PR TITLE
add skeleton change log file

### PR DESCRIPTION
**Pull request by mtbc**
Happy to adjust the format. Once approved, happy to spread further across https://github.com/ome/omero-python-superbuild.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# v5.6.dev1 (October 2019)
```
Comment (sbesson) [inline CHANGELOG.md]:
```suggestion
# 5.6.dev1 (October 2019)
```

Comment (mtbc) [reply CHANGELOG.md]: Thank you, will assume that's generally true across our Python decoupling.

**Comment by joshmoore** [comment]
I'll add some content to this before merging.

**Comment by joshmoore** [comment]
Merging for 5.6.dev3 (I'll add comments in the commit with the version bump).

**Pull request merged**